src/storage-client/src/controller.rs

```diff
@@ -467,6 +470,9 @@ impl RustType<ProtoCollectionMetadata> for CollectionMetadata {
                 .status_shard
                 .map(|s| s.parse().map_err(TryFromProtoError::InvalidShardId))
                 .transpose()?,
+            relation_desc: value
```

Comment:
what about the ones that already exist without this? do we need a migration of some sort?

Reply:
we don't need migrations for proto that goes over the grpc wire, its never stored!

Reply:
oh, TIL! (for some reason I'd thought this was in stash somewhere)